podcasts/New Search/Views/History/SearchHistoryCell.swift

```diff
@@ -0,0 +1,76 @@
+import SwiftUI
+import PocketCastsDataModel
+import PocketCastsUtils
+
+struct SearchHistoryCell: View {
+    @EnvironmentObject var theme: Theme
+
+    var podcast: Podcast?
+
+    var episode: Episode?
+
+    var searchTerm: String?
+
+    private var subtitle: String {
+        if let episode, let podcast {
+            let duration = TimeFormatter.shared.multipleUnitFormattedShortTime(time: TimeInterval(episode.duration))
+            return "\(L10n.episode) • \(duration) • \(podcast.title ?? "")"
+        } else if let podcast {
+            return "Podcast • \(podcast.author ?? "")"
```

Comment:
I did some testing where the author isn't returned and it to me it looked quite odd to have a trailing ` • `

<img width="399" alt="Screen Shot 2023-02-24 at 2 04 49 PM" src="https://user-images.githubusercontent.com/793774/221269115-e4819b6c-714c-4445-b168-5d5d5241c424.png">

wdyt about doing something like this?

```suggestion
            return [L10n.podcastSingular, podcast.author].compactMap { $0 }.joined(separator: " • ")
```

We can also use the `L10n.podcastSingular` here instead of the string directly.